src/plugins/plot/src/configuration/PlotSeries.js

```diff
@@ -377,13 +377,17 @@ define([
          * @public
          */
         updateFiltersAndRefresh: function (updatedFilters) {
-            this.filters = updatedFilters;
-            this.reset();
-            if (this.unsubscribe) {
-                this.unsubscribe();
-                delete this.unsubscribe;
+            if (this.filters) {
+                this.filters = updatedFilters;
```

Comment:
move `this.filters = updatedFilters;` out of both blocks? as used in both if and else.